code/scripting/lua.cpp

```diff
@@ -363,11 +363,13 @@ void script_state::OutputLuaMeta(FILE *fp)
 	//***Enumerations
 	fprintf(fp, "<dt id=\"Enumerations\"><h2>Enumerations</h2></dt>");
 	for (uint32_t i = 0; i < Num_enumerations; i++) {
-		//WMC - This is in case we ever want to add descriptions to enums.
-		//fprintf(fp, "<dd><dl><dt><b>%s</b></dt><dd>%s</dd></dl></dd>", Enumerations[i].name, Enumerations[i].desc);
-
-		//WMC - Otherwise, just use this.
+		// WMC - This is in case we ever want to add descriptions to enums.
+		// fprintf(fp, "<dd><dl><dt><b>%s</b></dt><dd>%s</dd></dl></dd>", Enumerations[i].name, Enumerations[i].desc);
+  
+		if (stricmp(Enumerations[i].name, "VM_EXTERNAL_CAMERA_LOCKED")) { 		// Cyborg17 -- Omit the deprecated flag
```

Comment:
Since it may be possible for more flags to be deprecated, I feel this bit here should be something more future proof, such as 
`if ((Enumerations[i].name == "VM_EXTERNAL_CAMERA_LOCKED"))
  // || (Enumerations[i].name == "Some_Deprecated_Flag))
{
  continue;
}`

Reply:
The equality operator doesn't play nice with the linux builds on github.  But using the continue statement is a good idea, so I'll separate it out.

Reply:
Note that the example is psuedocode, you should use whatever style of string comparison the rest of the file uses.